typing_requirements.txt

```diff
@@ -0,0 +1,2 @@
+mypy==0.770
```

Comment:
If you install these requirements, then the typing tests should run and pass for you locally (I hope).